build/npm/gyp/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "license": "MIT",
   "devDependencies": {
-    "node-gyp": "^8.4.1"
+    "node-gyp": "^9.3.1"
```

Comment:
What's this update for?

Reply:
This also depends on http-proxy-agent and https-proxy-agent (in the dependency tree) and this updates brings updated libraries. Updating node-gyp doesn't look could cause harm, so I updated it. Let me know if otherwise.

Reply:
Sounds good. I'm kind of surprised VS Code isn't keeping this up to date themselves actually - usually they're pretty good about that. But I would hope there aren't any issues (and may actually make things better, we see node-gyp issues a lot)